policyengine_us/variables/gov/states/la/tax/income/deductions/la_itemized_deductions.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class la_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana itemized deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://revenue.louisiana.gov/TaxForms/IT540iWEB(2022)D1.pdf"
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        itemizes = tax_unit("tax_unit_itemizes", period)
```

Comment:
need to add the amount of the federal itemized deduction ( we will use `itemized_deductions_less_salt`) - not just medical

Reply:
8A does nothing in the form - disregard